services/state/roster/roster.proto

```diff
@@ -3,6 +3,8 @@ syntax = "proto3";
 package com.hedera.hapi.node.state.roster;
 
 /*
+ * Hedera Network Services Protobuf
+ *
```

Comment:
We don't include these lines in new protobuf; they're only present in old files that followed an obsolete recommendation.